apps/admin_app/assets/js/views/loader.js

```diff
@@ -11,6 +11,8 @@ import ProductNewView from './product/new';
 import DashboardIndexView from './dashboard/index';
 import Shipping_policyIndexView from './shipping_policy/index'
 import Shipping_policyEditView from './shipping_policy/edit'
+import Stock_locationNewView from './stock_location/new'
+import Stock_locationEditView from './stock_location/edit'
```

Comment:
@jyotigautam May I recommend you make it a consistent naming and fix the old wrong naming styles. 

I expect `import ShippingPolicyIndexView from './shipping_policy/index'`. Make any other changes if required.